code/controllers/subsystem/vote.dm

```diff
@@ -213,6 +219,10 @@ SUBSYSTEM_DEF(vote)
 
 		reset()
 		switch(vote_type)
+			if(VOTE_TYPE_SUPERCHARGE)
+				if(i_have_the_power())
```

Comment:
So this only will cut out if the PA controller_box is active?

Reply:
Right, if the PA has been turned on we assume the station has power and the vote is skipped.